src/server/hset_family.h

```diff
@@ -29,9 +29,14 @@ class HSetFamily {
   static int32_t FieldExpireTime(const DbContext& db_context, const PrimeValue& pv,
                                  std::string_view field);
 
+  static std::vector<long> SetFieldsExpireTime(const OpArgs& op_args, PrimeValue& pv,
```

Comment:
our style guide requires that we pass output variables (pv) last and by pointer.

Reply:
Just to clarify, the signature should be like this?
```cpp
static std::vector<long> SetFieldsExpireTime(const OpArgs& op_args, uint32_t ttl_sec, std::string_view key,
                                               CmdArgList values, PrimeValue* pv)
```

Reply:
I think he wants:

```
static std::vector<long> SetFieldsExpireTime(const OpArgs& op_args, uint32_t ttl_sec, std::string_view key,
                                               CmdArgList values, PrimeValue* pv)
```

Notice last argument, it's not a `reference` but a `pointer`

Reply:
Also @NegatioN I am fine with the PR once this is addressed. The rest of my comments are NIT. 

LGTM! :rocket: